test/test_mpi.c

```diff
@@ -127,65 +131,77 @@ int xccl_mpi_create_comm(MPI_Comm comm, xccl_team_h *team) {
 
 int xccl_mpi_test_init(int argc, char **argv,
                        xccl_collective_cap_t coll_types) {
-    char *var;
-    int rank, size;
+    char     *var;
+    int      rank, size;
+    char     *tl, *saveptr;
+    uint64_t tls = 0;
+    uint64_t i;
+    int      j;
+    xccl_tl_id_t *tl_ids;
+    unsigned     tl_count;
 
     MPI_Init(&argc, &argv);
     MPI_Comm_rank(MPI_COMM_WORLD, &rank);
     MPI_Comm_size(MPI_COMM_WORLD, &size);
 
     /* Init xccl library */
     var = getenv("XCCL_TEST_TLS");
-    xccl_params_t params = {
-        .field_mask = XCCL_LIB_CONFIG_FIELD_TEAM_USAGE,
-        .team_usage = XCCL_USAGE_SW_COLLECTIVES |
-                      XCCL_USAGE_HW_COLLECTIVES,
+    if (var) {
```

Comment:
Maybe inline function in xccl_tls.h tls_string_to_bitmap ? It is also used somewhere inside xccl code if i recall.